Slambook/index.js

```diff
@@ -10,12 +10,9 @@ document.querySelector('.form').addEventListener('submit', function(event) {
     event.preventDefault(); // Prevent form submission
 
     // Log to console when submit event occurs

```

Comment:
Consider adding error handling if the query selector does not find an element with the name 'contactName'.